test/unit/messaging/process-return-message.test.js

```diff
@@ -15,9 +17,7 @@ describe('process return message', () => {
 
   test('completes message', async () => {
     const message = {
-      body: {
-        invoiceNumber: '12345'
-      }
+      body: { ...settlement }
     }
     await processReturnMessage(message, receiver)
     expect(receiver.completeMessage).toHaveBeenCalledWith(message)
```

Comment:
missing tests in this file now -- use this as a copy-pasta reference: https://github.com/DEFRA/ffc-pay-statement-constructor/pull/4/files#diff-10b5256f2669b5274de35c98126911d1a861a5de2e6fb01ca93d8cea7ff84023L1